src/content/docs/distribute/rpm.mdx

```diff
@@ -0,0 +1,397 @@
+
+- **conflict**: Prevents the installation of the package if it conflicts with another package.
+  For example, if you update an RPM package that your app depends on and the new version is incompatible with your app.
+
+- **provides**: Lists the RPM dependencies that your application provides.
+
+- **depends**: Lists the RPM dependencies that your application needs to run.
+
+- **files**: Specifies which files to include in the package.
+
+- **obsoletes**: Lists the RPM dependencies that your application obsoletes.
+
+:::note
+If this package is installed, packages listed as “obsoletes” will be automatically removed
+if present.
+:::
+
+- **desktopTemplate**: Adds a custom desktop file to the package.
+
+- **epoch**: Defines weighted dependencies based on version numbers.
+
+:::caution
+It is not recommended to use epoch unless necessary, as it alters how the package manager compares package versions.
+For more information about epoch, please check: [RPM Packaging Guide](https://rpm-packaging-guide.github.io/#epoch-scriptlets-and-triggers).
+:::
+
+To use these options, add the following to your `tauri.conf.json` :
+
+```json title="tauri.conf.json"
+{
+  "productName": "tauri-app",
+  "version": "0.0.0",
+  "identifier": "com.tauri.devthom",
+  "build": {
+    "beforeDevCommand": "npm run dev",
+    "devUrl": "http://localhost:1420",
+    "beforeBuildCommand": "npm run build",
+    "frontendDist": "../dist"
+  },
+  "app": {
+    "withGlobalTauri": true,
+    "windows": [
+      {
+        "title": "tauri-app",
+        "width": 800,
+        "height": 600
+      }
+    ],
+    "security": {
+      "csp": null
+    }
+  },
+  "bundle": {
+    "active": true,
+    "targets": "all",
+    "icon": [
+      "icons/32x32.png",
+      "icons/128x128.png",
+      "icons/128x128@2x.png",
+      "icons/icon.icns",
+      "icons/icon.ico"
+    ],
+    "category": "Utility",
+    "linux": {
+      "rpm": {
+        "epoch": 0,
+        "files": {},
+        "release": "1",
+          "preInstallScript": "/path/to/your/project/src-tauri/scripts/prescript.sh",
+        "postInstallScript": "/path/to/your/project/src-tauri/scripts/postscript.sh",
+        "preRemoveScript": "/path/to/your/project/src-tauri/scripts/prescript.sh",
+        "postRemoveScript": "/path/to/your/project/src-tauri/scripts/postscript.sh"
+        "conflicts": ["oldLib.rpm"],
+        "depends": ["newLib.rpm"],
+        "obsoletes": ["veryoldLib.rpm"],
+        "provides": ["coolLib.rpm"],
+        "desktopTemplate": "/path/to/your/project/src-tauri/desktop-template.desktop"
+      }
+    }
+  }
+}
+```
+
+### Add a license to the package
+
+To add a license to the package, add the following to the `src-tauri/cargo.toml` file:
+
+```toml title="src-tauri/cargo.toml"
+[package]
+name = "tauri-app"
+version = "0.0.0"
+description = "A Tauri App"
+authors = ["you"]
+edition = "2021"
+license = "MIT" # add the license here
```

Comment:
If we mention this, i think we should also mention bundle.license and/or bundle.licenseFile in tauri.conf.json 🤔 what do you think?

Reply:
yeah that make sense to mention both  to show bundle.license  but for bundle.licenseFile  maybe but i do not know the difference with bundle.license. But  for what i understand this will find the license file if you have multiple config file like `tauri.linux.conf.json`.    I have included it  anyway in the rpm doc

Reply:
bundle.license is the same as license in cargo.toml for those who don't want to touch the cargo.toml file.
bundle.licenseFile is a seperate license file that some bundles will include (not sure which ones, may be windows-only). What you did is good enough imo